### DISCUSSION TITLE
Does anyone know how to make a parallel coordinates chart with vx?

### DISCUSSION THREAD

**Discussion by kevinxia787** [airbnb/visx#1927] [General]

**Comment by williaster** [comment]
@kevinxia787 I haven't seen an example of this yet but should definitely be possible (tho some parallel coordinates visualizations are implmented with `<canvas />` for perf, which vx does not currently support).

I'll work on this next time I have a bit of free time unless someone beats me to it.

**Comment by techniq** [comment]
@kevinxia787 @williaster Coincidently I've been looking at these / bump charts lately (but haven't attempted an example yet).

Here are some examples I've seen that might be useful for reference:

- http://www.columbia.edu/~cme2126/datavisuals/bigdata_parallelcoordinates.html
- [Exoplanets II](https://bl.ocks.org/syntagmatic/482706e0638c67836d94b20f0cb37122)
- [Nutrient Parallel Coordinates IV](https://bl.ocks.org/syntagmatic/05a5b0897a48890133beb59c815bd953)
- https://syntagmatic.github.io/parallel-coordinates/
- https://bl.ocks.org/jasondavies/1341281
- https://github.com/syntagmatic/parallel-coordinates
- [Nutrient Parallel Coordinates](http://bl.ocks.org/syntagmatic/3150059)
- http://blockbuilder.org/eesur/1a2514440351ec22f176
- https://formidable.com/open-source/victory/gallery/parallel-brush-axis/
- [Slopegraphs](http://charliepark.org/slopegraphs/)

Can also be similar to some "bump charts" I've seen
- https://bl.ocks.org/johnwalley/a0734cf335b44365026adae40cce5945
- http://bl.ocks.org/cjhin/b7a5f24a0853524414b06124c559961a

although I think of bump charts more like:
- https://roadtolarissa.com/stacked-bump/

Some possible features in demo
- Brushable axises (:cough @hshoff :cough)
- Draggble axises (see: [Parallel Coordinates Plot in plotly.js](https://plot.ly/javascript/parallel-coordinates-plot/))
- ...

**Comment by m-sterspace** [comment]
Sorry to revise this from the grave, but out of curiosity has anyone found a good way of implementing a parallel coordinates plot using VisX?
